src/ui/popup/theme/controls/reset-button.tsx

```diff
@@ -0,0 +1,29 @@
+import {m} from 'malevic';
+import {Button} from '../../../controls';
+import ControlGroup from '../../control-group';
+import {ViewProps} from '../../types';
+import {DEFAULT_SETTINGS} from '../../../../background/user-storage';
+
+export default function ResetButton(props: ViewProps) {
+    function reset() {
+        props.actions.setTheme(DEFAULT_SETTINGS.theme);
+        props.actions.changeSettings({customThemes: []});
```

Comment:
I think this line should be removed. `props.actions.setTheme()` changes the current theme preset (default or custom).

Reply:
FilterConfig?

Reply:
theme: FilterConfig

defined in the src\definitions.d.ts

Reply:
![](https://i.imgur.com/JHXO4NE.png) Just the FilterConfig...

Reply:
It is planned to rename `FilterConfig` into `Theme`.

Reply:
Let's remove `props.actions.changeSettings({customThemes: []});` because it resets all custom settings for all websites.

Reply:
'Reset this values` in the FilterConfig page will need to reset theme..